Project.toml

```
@@ -92,7 +92,7 @@ StaticArrays = "1.5"
 StrideArrays = "0.1.26"
 StructArrays = "0.6.11"
 SummationByPartsOperators = "0.5.41"
-T8code = "0.4.3, 0.5"
+T8code = "0.4.3, 0.5, 0.6"
```

```suggestion
T8code = "0.5, 0.6"
```
[This error](https://github.com/trixi-framework/Trixi.jl/actions/runs/8845696009/job/24290125584?pr=1900#step:8:4758) in the Downgrade action looks like we use features from T8code.jl v0.5 and thus Trixi.jl is not compatible with T8code.jl v0.4.3, right?

---

Restricted to `0.5` for this PR.